crates/commitlog/src/tests/helpers.rs

```diff
@@ -38,6 +38,17 @@ where
     total_txs
 }
 
+pub fn fill_log_with<R, T>(log: &mut commitlog::Generic<R, T>, txes: impl IntoIterator<Item = T>)
```

Comment:
I think this should at least have commentary which explains that `txes` must fit into a single commit as per the options of the `commitlog::Generic` being passed in.
And perhaps that it will create a single commit.